packages/features/ee/payments/api/webhook.ts

```diff
@@ -34,6 +34,91 @@ async function getEventType(id: number) {
   });
 }
 
+async function getBooking(bookingId: number) {
```

Comment:
DRY again. Will reuse this logic for `payment_intent.succeeded` and `setup_intent.succeeded` events